inst/rmd/h/anchor-sections/anchor-sections-icon.css

```diff
@@ -0,0 +1,7 @@
+/* Styles for section anchors content */
+/* From https://material.io/resources/icons/
+   Licence: https://www.apache.org/licenses/LICENSE-2.0.html */
+
+a.anchor-section:before {
+  content: url(./link-black-24dp.svg);
```

Comment:
Is it worth inlining the svg here?

Reply:
Do you mean not have it as an external file but directly put its content in one line i.e `<svg>...</svg>` ?

Just to be sure of the meaning on "inlining"

Reply:
Well, using a data url, but yes — include all of the svg data in the css file to avoid having a separate file.